docs/dev/table/streaming.md

```diff
@@ -22,21 +22,166 @@ specific language governing permissions and limitations
+
+In the following we show two example queries on a `clicks` table that is defined on the stream of click events.
+
+The first query is a simple `GROUP-BY COUNT` aggregation query. It groups the `clicks` table on the `user` field and counts the number of visited URLs. The following figure shows how the query is evaluated over time as the `clicks` table is updated with additional rows.
+
+<center>
+<img alt="Continuous Non-Windowed Query" src="{{ site.baseurl }}/fig/table-streaming/query-groupBy-cnt.png" width="90%">
+</center>
+
+The input table `clicks` is shown on the left-hand side. Initially, the table consists of six rows. Evaluating the query (shown in the middle) on these six records yields a result table which is shown on the right-hand side at the top. When the `clicks` table is updated by appending an additional row (originating from the stream of click events), the query updates the current result table and increases the appropriate count. The updated result table is show on the right-hand side in the middle (the updated row is highlighted). Finally, another row is added and the result is shown on the right bottom of the figure.
+
+The second query is similar to the first one but groups the `clicks` table in addition to the `user` attribute also on an [hourly tumbling window](./sql.html#group-windows) before it counts the number of URLs. Again, the figure shows the input and output at different points in time to visualize the changing nature of dynamic tables.
+
+<center>
+<img alt="Continuous Group-Window Query" src="{{ site.baseurl }}/fig/table-streaming/query-groupBy-window-cnt.png" width="100%">
+</center>
+
+The input table `clicks` is shown on the left. The query continuously computes results every hour and updates the result table. The clicks table contains four rows with timestamps (`cTime`) between `12:00:00` and `12:59:59`. The query computes two results rows from this input (one for each `user`) and appends them to the result table. For the next window between `13:00:00` and `13:59:59`, the `clicks` table contains three rows, which results in another two rows being appended to the result table. As more records arrive over time, the result table is appropriately updated.
+
+**Note:** Time-based computations such as windows are based on special [Time Attributes](#time-attributes), which are discussed below.
+
+#### Update and Append Queries
+
+Although the two example queries appear to be quite similar (both compute a grouped count aggregate), they differ in one important aspect. The first query must update previously emitted results, i.e., the changelog stream that defines the result table contains `INSERT` and `UPDATE` changes. In contrast, the second query only appends to the result table, i.e., the changelog stream of the result table consists only of `INSERT` changes.
```

Comment:
Although the two example queries appear to be quite similar (both compute a grouped count aggregate), they differ in one important aspect, i.e., ... What do you think?